### DISCUSSION TITLE
Can’t read Shuttermode

### DISCUSSION THREAD

**Discussion by MMeinhardt1** [arendst/Tasmota#15795] [Q&A] [answered]
Hi everybody,

in the console i can't see the defined ShutterMode, see below. 

```
21:00:54 CMD: Shuttermode
21:00:54 MQT: stat/Jalousie_Bad/RESULT = {"Command":"Unknown"}
21:01:12 CMD: Setoption80
21:01:12 MQT: stat/Jalousie_Bad/RESULT = {"SetOption80":"ON"}
```

What goes wrong or it is normal?

Many thanks

**Comment by sfromis** [comment, marked as answer]
You may be running a binary not including the Shutters feature. With standard precompiled `tasmota.bin`, the command `ShutterMode` should indeed be available after having `SetOption80 ON`.

**Comment by MMeinhardt1** [comment]
Hi @sfromis ,
in my post you can see that the "SetOption80" is ON?

So we should assume that the binary was compiled with appropriate ShutterMode. The slider can also be seen on the start page of Tasmota and also work. All other "ShutterMode" functions also work very well, this should only be available if a corresponding binary has been compiled with ShutterControl.

**Reply by sfromis** [reply]
`SetOption80` being on means nothing about what is included in the binary. The reason for just having an option number is that the options are simply a long bit array, where any bit can be on or off irrespective of related features being available.

**Comment by MMeinhardt1** [comment]
Hi,
i think the slider is only available if the binary is compiled with the shutter function. I can see the slider. Is this right and what's going wrong?
![image](https://user-images.githubusercontent.com/56205459/173895693-7b9f744d-4164-4a48-83a0-78ef8aa53320.jpeg)

**Reply by sfromis** [reply]
What I meant was that what you originally posted had no clue to suggest if you had shutter support included or not, as the availability of `SetOption80` does not indicate anything at all. 

Now you probably inadvertently provided a relevant clue about the situation, that you are using a very outdated Tasmota version, 8.3.1. Do not expect that current docs will apply to that old builds. The command ShutterMode was only introduced in version 8.5.1

**Reply by MMeinhardt1** [reply]
Ok, i will compile a new version in the next days and then i test it again. In my opinion the SetOption80 was a indicator of the shutter-compiled binary. Sorry for my mistake.
